olp-cpp-sdk-authentication/src/TokenResult.h

```diff
@@ -19,14 +19,12 @@
 
 #pragma once
 
-#include <olp/core/porting/deprecated.h>
-
 #include <chrono>
 #include <ctime>
 #include <string>
 
-#include "AuthenticationApi.h"
-#include "ErrorResponse.h"
+#include <olp/authentication/AuthenticationApi.h>
+#include <olp/authentication/ErrorResponse.h>
 
 namespace olp {
 namespace authentication {
```

Comment:
Cannot add a suggestion for unchanged lines :(
class TokenResult must be without AUTHENTICATION_API

Reply:
done